.gitleaks.toml

```diff
@@ -0,0 +1,11 @@
+[allowlist]
```

Comment:
Do we still need to introduce this file?

Reply:
removed